DancingGoat/Areas/Admin/Controllers/SelfConfigController.cs

```diff
@@ -29,13 +30,19 @@ public class SelfConfigController : Controller
         public const int PROJECT_EXISTENCE_VERIFICATION_REQUIRED_ITEMS = 32;
 
         protected readonly SelfConfigManager _selfConfigManager;
-        protected readonly IDeliveryClient client = new SampleDeliveryClient();
+        protected IDeliveryClient client { get; private set; }
```

Comment:
inconsistency in naming: do we prefix protected with underscore or not? (_selfConfigManager vs client)

Reply:
IDK - eg.: 'protected baseClient' without underscore.

Reply:
Fixed

Reply:
Fixed to `SelfConfigManager `and `Client`.